sql/moz-fx-data-shared-prod/contextual_services/event_aggregates_spons_tiles/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.contextual_services.event_aggregates_spons_tiles`
+AS
+SELECT
+  * REPLACE (LOWER(advertiser) AS advertiser)
```

Comment:
Is there a reason we don't normalize the advertiser in the derived dataset?

Reply:
ah okay - so we may need to preserve the original capitalization for use cases where we share the data with partners. Since this is internal use, I will move this lower-casing from the view.sql file to the query.sql file.